README.md

```diff
@@ -82,32 +83,54 @@ More on the [Git workflow for this project](docs/git_workflow.md).
 
 ### Requirements
 
-* PHP 7.4
+* PHP 7.4 (Upgrading to 8.2)
 * [Composer](https://getcomposer.org/)
 * [Symfony CLI](https://symfony.com/download#step-1-install-symfony-cli)
 
 ### Clone the repository
 
 `git clone git@github.com:w3c/website-frontend.git`
 
-Run `composer install`
+### Install and update composer dependencies
+
+```bash
+composer install
+
+#Once the install is done
+
+composer update
+```
+
+***Note:*** If you already have this project installed locally and you're having trouble seeing any changes, make sure you have cleared your Symfony cache using the `bin/console cache:clear` command.
 
 ### Configuration
 Create a local environment file and populate the required variables (see `.env.local.dist`):
 
 ```
 cp .env.local.dist .env.local
 ```
+**Configure your local env file**
+The W3C API URL and the Craft API URL are available in 1pass.
```

Comment:
1pass is only relevant to S24 so I suggest we remove that sentence. Those two variables are already pre-configured in .env so it shouldn't be necessary to change them.

Note that I don't think having .env.local.dist is very useful as .env already contains defaults for most variables, only secret ones (and probably APP_ENV need to be set in .env.local)

Reply:
I can review which settings we definitely need in the local env file and which are non-sensitive and can be moved to the environment env file (.env.dev, .env.prod which doesn't exist yet). 

A .dist file is useful to document which settings are required in a local env file. These should really only be secrets.

Reply:
Rather that creating and maintaining a .dist file I'd rather say that all variables that are in .env can be overidden in .env.local as necessary, in particular secret and password variables (but also cms url if you run one locally for example). Symfony [stopped  using .dist files](https://symfony.com/doc/4.1/configuration/dot-env-changes.html) for environment variables with Sf4.1 in November 2018. Since then, .env is the new .env.dist.

Reply:
We don't use .dist in Symfony itself, rather it's a convention to help us remember what we need in a local env file. 

But I see what you are getting at & I can remove this and update the installation docs.